ui/panels-plugin/package.json

```diff
@@ -50,6 +50,8 @@
     "plugin.json"
   ],
   "devDependencies": {
+    "@perses-dev/internal-utils": "0.29.1",
+    "@perses-dev/plugin-system": "0.29.1",
```

Comment:
Does @perses-dev/plugin-system need to be added here since it's already in `dependencies`?

Reply:
It shouldn't. Mistake on my part trying to fix some type errors. Will fix that.